Mathlib/Algebra/Order/Interval/Finset/SuccPred.lean

```diff
@@ -0,0 +1,102 @@
+/-
+Copyright (c) 2021 Yaël Dillies. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies
+-/
+import Mathlib.Algebra.Group.Embedding
+import Mathlib.Algebra.Order.Interval.Set.Monoid
+import Mathlib.Algebra.Order.Interval.Finset.Basic
+import Mathlib.Algebra.Order.SuccPred
+import Mathlib.Order.Interval.Finset.SuccPred
+
+/-!
+# Finset intervals in a successor-predecessor order
+
+This files proves relations between the various finset intervals in a successor/predecessor order.
```

Comment:
Let's make sure the docstring here isn't an exact copy of that in another file: it's a pretty bad sign for two different files to have identical docstrings! In addition, it would be nice to describe the distinction between the lemmas that should be in this file vs those that should be in the other Order/Interval/Finset/SuccPred file.

Reply:
Whoops, copy pasta. What about now?